packages/manager/src/features/CloudPulse/Alerts/CreateAlert/NotificationChannel/RenderChannelDetails.tsx

```diff
@@ -0,0 +1,23 @@
+import { Chip } from '@mui/material';
+import * as React from 'react';
+
+import type { NotificationChannel } from '@linode/api-v4';
+
+interface RenderChannelDetailProps {
+  /**
+   * Notification Channel with the data to be shown in the component
+   */
+  template: NotificationChannel;
+}
+export const RenderChannelDetails = (props: RenderChannelDetailProps) => {
+  const { template } = props;
+  if (template.channel_type === 'email') {
+    return (
+      template.channel_type === 'email' &&
```

Comment:
why dual condition check? We have already put the check inside `if`

Reply:
The second condition implicitly casts the NotificationChannel into NotificationChannelEmail which has different `content` json object , so to use `template.email.content` I'm doing the condition check again.

Reply:
extra condition is not required you remove that 

```suggestion
if (template.channel_type === 'email') {
    return template.content.email.email_addresses.map(
      (email: string, id: number) => <Chip key={id} label={email} />
    );
  }
```